crates/store/src/genesis.rs

```diff
@@ -60,7 +61,7 @@ impl GenesisState {
             Smt::default().root(),
             *EmptySubtreeRoots::entry(BLOCK_NOTE_TREE_DEPTH, 0),
             Digest::default(),
-            Digest::default(),
```

Comment:
Faucet was failing to execute correctly due to this being an invalid kernel root.